rest-server/server/src/main/resources/reference.conf

```diff
@@ -34,5 +34,6 @@ elasticsearch {
 }
 
 spark {
-  eventLog.dir="file:///tmp"
+  eventLog.dir = "file:///tmp"
+  job-server-url = ${SPARK_JOB_SERVER_URL}
```

Comment:
I would really like to stop using akka conf. If SPARK_JOB_SERVER_URL is changed elsewhere this might cause a bug. Why not just use the env to get these values instead of the syntactic sugar of akka conf. We know they have to be in the env anyway so this is not DRY to duplicate in akka conf.

Reply:
spark.job-server-url is an app level setting, ${SPARK_JOB_SERVER_URL} is an env level one.
with akka.conf we have one place for mapping...

Reply:
and it should be `harness-env` not .conf files spread around everywhere. Since we have to use harness-env it should be all we use, no?

Reply:
I see you point about being dry I still don't like having 2 mechanisms from reading env into the code. Why not ONLY have `harness-env`